src/cargo/core/compiler/build_context/target_info.rs

```diff
@@ -519,7 +519,7 @@ fn parse_crate_type(
 }
 
 /// Helper for creating an error message when parsing rustc output fails.
-fn output_err_info(cmd: &ProcessBuilder, stdout: &str, stderr: &str) -> String {
+pub fn output_err_info(cmd: &ProcessBuilder, stdout: &str, stderr: &str) -> String {
```

Comment:
I think this may no longer be needed

Reply:
You are correct. It can be reverted back to private scope.